# Mapper params
scan_resolution: 0.5
map_resolution: 0.5
neighbor_radius: 20.0
max_neighbor_links: 1
patch_building_range: 4
map_outlier_radius: 0.2
map_outlier_neighbors: 0
min_translation: 1.0
min_rotation: 1.0

use_odometry: true
add_odometry_edges: true
use_odometry_heading: false

robot_name: "Mia"
odometry_frame: "odom"
robot_frame: "base_link"
map_frame: "map"
laser_frame: "velodyne"

# GICP params
icp_fine/max_correspondence_distance: 2.0
icp_fine/max_fitness_score: 50.0
icp_fine/maximum_iterations: 50
icp_fine/point_cloud_density: 0.5

icp_coarse/max_correspondence_distance: 5.0
icp_coarse/max_fitness_score: 25.0
icp_coarse/maximum_iterations: 50
icp_coarse/point_cloud_density: 2.0
